src/linearmixedmodel.jl

```diff
@@ -213,10 +213,21 @@ function StatsAPI.fit(::Type{LinearMixedModel},
     σ=nothing,
     amalgamate=true,
     kwargs...)
-    lmm = LinearMixedModel(f, tbl; contrasts, wts, σ, amalgamate)
-    return fit!(lmm; kwargs...)
+    lmod = LinearMixedModel(f, tbl; contrasts, wts, σ, amalgamate)
+    return fit!(lmod; kwargs...)
 end
 
+"""
+    lmm(f::FormulaTerm, tbl; kwargs...)
+    lmm(
+        f::FormulaTerm, tbl;
+        wts=[], contrasts=Dict{Symbol,Any}(), σ=nothing, amalgamate=true,
+        kwargs...)
+
+    Aliases for the fit(LinearMixedModel, ...) functions
+"""
```

Comment:
```suggestion
"""
    lmm(args...; kwargs...)

Convenience wrapper for `fit(LinearMixedModel, args...; kwargs...)`.

See [`fit`](@ref) for more information.
"""
```

Then I would add an example somewhere in the narrative docs, maybe in the introductory sections.

Reply:
thanks!